wiggum_app/models.py

```diff
@@ -87,15 +87,17 @@ def checkSameMetadata(labeled_df, meta):
 
     return checkResult
 
-def getDistanceHeatmapDict(df):
+def getDistanceHeatmapDict(labeled_df, df):
```

Comment:
this is redundant to take both the labeled dataframe and the result_df separately, just pass the labeled_df.

Reply:
change line 108 (and other usses of df) to labeled_df.result_df

Reply:
The df parameter will be passed by filter_df, detect_df and rank_df, not always result_df.

Reply:
```suggestion
def getDistanceHeatmapDict(labeled_df, cur_result_df):
```
change the name to be more explicit

Reply:
Changed